backends/edk2-compat/edk2-svc-generate.c

```diff
@@ -6,10 +6,10 @@
 #include <ctype.h> // for isspace
 #include <mbedtls/md.h>     /* generic interface */
 #include <mbedtls/platform.h> /*mbedtls functions*/
-#include "../../extraMbedtls/include/pkcs7.h" // for PKCS7 OID
-#include "include/endian.h"
+#include "../../external/extraMbedtls/include/pkcs7.h" // for PKCS7 OID
+#include "../../external/skiboot/include/endian.h"
```

Comment:
Maybe add `-I external/` to the compiler flags, so all of these can be changed to `#include <skiboot/include/endian.h>`?

Alternatively, `-I .` (if not already set) would work so that all headers are preceded with `external/`, to make it clear its coming from somewhere else.

Either way, these long relative paths can get a little noisy

Reply:
I kind of like them as is. I think for clarity it is nice to see what headers are part of the package and which ones are expected to be on the system already. Maybe, I am just being lazy but let me know if this needs to be changed

Reply:
Leave it for now, but this should probably get cleaned up. I think it might help keep the external code completely unchanged, since we wouldn't have to modify header paths.